api/endpoints/move.md

```diff
@@ -0,0 +1,297 @@
+                        "id": 81,
+                        "content": "new comment here!"
+                    }
+                ],
+                "dateCreated": 1469657150797,
+                "id": 4,
+                "dateUpdated": 1488473461303,
+                "status": {
+                    "code": "apr",
+                    "dateCreated": 1195932197030,
+                    "name": "Approved by Manager",
+                    "id": 14
+                }
+           }
+
+
+
+### Remove a Move Request [DELETE]
++ Parameters
+    + id (string) ... ID of the User
++ Response 200
+
+           {
+                "response": "Successfully removed"
+           }
+
+
+## Move Collection [/move/requests]
+
+List of all Move Requests
+
++ Model (application/json)
+    JSON representation of the Move Collection resource.
+
+    + Body
+
+            [
+              {
+                "comments": [
+                    {
+                      "id": 47,
+                      "content": "Word."
+                    }
+                ],
+                "dateCreated": 1469657281923,
+                "id": 6,
+                "status": {
+                  "code": "sub",
+                  "dateCreated": 1185984126540,
+                  "name": "In Progress",
+                  "id": 10
+                }
+              },
+              {
+                "comments": [
+                  {
+                    "id": 48,
+                    "content": "Word 2 ur mother."
+                  }
+                ],
+                "dateCreated": 1469657337973,
+                "id": 7,
+                "status": {
+                  "code": "sub",
+                  "dateCreated": 1185984126540,
+                  "name": "Pending",
+                  "id": 15
+                }
+              },
+              {
+                "comments": [
+                  {
+                    "id": 51,
+                    "content": "Word 'em up!"
+                  }
+                ],
+                "dateCreated": 1469657800263,
+                "id": 8,
+                "status": {
+                  "code": "sub",
+                  "dateCreated": 1185984126540,
+                  "name": "Approved by Manager",
+                  "id": 14
+                }
+              },
+            ]
+
+
+### List of all Move Requests [GET]
+
++ Response 200
+
+    [Move Collection][]
+
+
+### Create a Move Request [POST /move/requests]
+The following attribute is required to update a Move Request: status `id`.
+
++ Request (application/json)
+
+             {
+                "comments": [
+             {
+                "content": "this is a new move request"
+             }
+                ],
+                "status": {
+                "id": 13
+                }
+            }
+
++ Response 201
+
+
+### Get Approval of a Move Request [GET /move/requests//{{id}//?selector=approved]
+
+Get boolean value if Move Request {id} is approved
```

Comment:
This gives you the approved value of a specific move request. It will return true or false (not just if the move request is approved).